### DISCUSSION TITLE
Security Onion Machine Date and Time update Issue

### DISCUSSION THREAD

**Discussion by usmanaa10** [Security-Onion-Solutions/securityonion#11006] [Unsupported Versions] [answered]
Hi Fellows,
           I am facing an issue with the security onion machine date and time, I update the date and time using **timedateclt** feature to local time/ zone but whenever I restart the SO system its local time again changes to UTC as shown in the picture below

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/66419518/2b30aa81-7d76-4cf8-be99-bb8ff1a7a3f7)

Can anyone help me how to keep the security onion machine time and date locally even if it restarts

Thanks

**Comment by TOoSmOotH** [comment, marked as answer]
https://docs.securityonion.net/en/2.3/timezones.html

**Reply by usmanaa10** [reply]
Thanks
